tests/components/openuv/test_config_flow.py

```diff
@@ -30,6 +28,9 @@
 
 async def test_invalid_api_key(hass):
     """Test that an invalid API key throws an error."""
+    import pyopenuv
```

Comment:
If we have added this library to test requirements, imports should go at the top of the module.